angular-cli.json

```diff
@@ -50,6 +59,17 @@
   "defaults": {
     "styleExt": "scss",
     "prefixInterfaces": false,
-    "lazyRoutePrefix": "+"
```

Comment:
I did have one question. `lazyRoutePrefix` is deprecated in the CLI config & `./src/app/users/+form` is named like it's supposed to be lazy loaded though the route config isn't loading it that way.

Which way do you want it?